### DISCUSSION TITLE
Supprimer plusieurs scènes à la fois

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#594] [Ideas]
Bonjour,

Serai t'il possible d'avoir la possibilité de supprimer plusieurs scènes en même temps.

On sélectionne toutes les scènes que l'on veut supprimer en même temps puis on clique sur un bouton de suppression des scènes sélectionnés.
L'ajout d'une fenêtre qui demande une confirmation de suppression pourrait être un plus pour éviter une erreur dans les suppressions.

Quand on fait un modèle d'une programmation pour un nouveau projet. Si la programmation d'origine a une multitute de scène, c'est assez long de supprimer tout ce qui n'est pas nécessaire pour le nouveau projet.

Merci d'avance ;)

**Comment by Raphwit** [comment]
C'est une bonne idée et je comprend l'utilité d'une telle fonctionnalité mais puisque la suppression de scène ne dispose pas de retour en arrière, une mauvaise sélection (même avec confirmation) pourrait entrainer la perte de quelques heures de travail si aucune sauvegarde n'a été effectuée au préalable.

Nous réfléchissons donc à une telle fonctionnalité, merci de la suggestion cependant !

**Reply by JeremieDENAEGEL** [reply]
On pourrait peut-etre créer un dossier "Corbeille" où partirait les scènes supprimées.
Le dossier ne sera pas supprimable et ainsi le programmateur devra valider la suppression définitive des scènes dans ce dossier.
Ca pourrait pas servir à éviter des erreurs ?
